src/lib/__tests__/generateChartImg.test.js

```diff
@@ -0,0 +1,104 @@
+const generateChartImg = require('../generateChartImg');
+
+describe('generateChartImg function', () => {
+  test('returns a URL representing the chart image', async () => {
+    // Define input parameters
+    const type = 'line';
+    const data = [{ x: 1, y: 2 }, { x: 2, y: 3 }, { x: 3, y: 4 }];
+    const xLabel = 'X Axis';
+    const yLabel = 'Y Axis';
+    const title = 'Chart Title';
+    const color = 'blue';
+
+    // Call the generateChartImg function
+    const imgUrl = await generateChartImg(type, data, xLabel, yLabel, title, color);
+
+    // Check if the function returns a URL representing the chart image
+    expect(imgUrl).toMatch(/^blob:/);
```

Comment:
Maybe add a more detailed comment to specify what "/^blob:/" does. Just a small recommendation as the test look really good